SQL/0000-00-00-schema.sql

```diff
@@ -117,6 +117,13 @@ DROP TABLE IF EXISTS `mri_scanner`;
 DROP TABLE IF EXISTS `mri_processing_protocol`;
 DROP TABLE IF EXISTS `ImagingFileTypes`;
 
+DROP TABLE IF EXISTS `tarchive_files`;
+DROP TABLE IF EXISTS `tarchive_series`;
+DROP TABLE IF EXISTS `tarchive`;
+
+DROP TABLE IF EXISTS BIDS_mri_scan_type_rel;
```

Comment:
table names should be in loweracase according to the LORIS SQL standard (https://github.com/aces/Loris/blob/minor/docs/SQLModelingStandard.md)

Reply:
I changed it to lowercase in the last commit; so we have both versions, depending on what @PapillonMcGill decides